src/contracts/StakedTokenV3.sol

```diff
@@ -512,7 +529,7 @@ contract StakedTokenV3 is
       } else {
         stakersCooldowns[from].amount =
           stakersCooldowns[from].amount -
-          amountToRedeem.toUint184();
+          uint104(amountToRedeem);
```

Comment:
noticed we already safecast to 104 a few lines above: https://github.com/bgd-labs/aave-stk-gov-v3/pull/13/files#diff-9442693c9eec991e258578e5750be9e613d52a131c187d6783f11766efd4ebcdR524